src/main/java/org/fundacion/jala/converter/models/facade/ZipFileFacade.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Zacarias Huanca Alconz
+ */
+package org.fundacion.jala.converter.models.facade;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import static org.fundacion.jala.converter.models.AssetSQL.insertAssetData;
+import static org.fundacion.jala.converter.service.ZipService.zipFile;
+import static org.fundacion.jala.converter.service.ZipService.zipFiles;
+
+public class ZipFileFacade {
+
+    private ZipFileFacade() {
+    }
+
+    /**
+     * Compress files of Audio
```

Comment:
It should be written in the third person.

Reply:
ok